templates/course-theme/single-lesson.php

```diff
@@ -19,18 +19,29 @@
 }
 ?>
 
-<!-- wp:group {"className":"sensei-course-theme__header"} -->
-<div class="wp-block-group sensei-course-theme__header sensei-course-theme__frame">
-	<!-- wp:sensei-lms/course-theme-course-progress-counter /-->
-
-	<!-- wp:sensei-lms/course-theme-prev-next-lesson -->
-	<!-- wp:sensei-lms/course-theme-prev-lesson {"inContainer":true} /-->
-	<!-- wp:sensei-lms/course-theme-next-lesson {"inContainer":true} /-->
-	<!-- /wp:sensei-lms/course-theme-prev-next-lesson /-->
+<!-- wp:sensei-lms/container {"className":"sensei-course-theme__header"} -->
```

Comment:
Is this something we still need to implement? Do you think we really need a specific block here instead of a group block?

Reply:
Right, I added this as something inert because the Group block was bringing in some messy CSS for its content. So it'll depend on where we end up with theme styles. If we need to keep this we can set it up properly.